diff --git a/glib/tests/utils.c b/glib/tests/utils.c
index 0441cd3..69a0d5c 100644
--- a/glib/tests/utils.c
+++ b/glib/tests/utils.c
@@ -440,7 +440,7 @@ test_xdg_dirs (void)
 
   xdg = (gchar *)g_getenv ("XDG_CONFIG_DIRS");
   if (!xdg)
-    xdg = "/etc/xdg";
+    xdg = "/usr/share/xdg" G_SEARCHPATH_SEPARATOR_S "/etc/xdg";
 
   dirs = g_get_system_config_dirs ();
 
--- glib-2.60.0/glib/gutils.c~	2019-03-04 20:06:14.000000000 +0000
+++ glib-2.60.0/glib/gutils.c	2019-03-10 23:14:11.808855876 +0000
@@ -2174,7 +2174,7 @@
     }
 #else
   if (!conf_dirs || !conf_dirs[0])
-    conf_dirs = "/etc/xdg";
+    conf_dirs = "/usr/share/xdg" G_SEARCHPATH_SEPARATOR_S "/etc/xdg";
 
   conf_dir_vector = g_strsplit (conf_dirs, G_SEARCHPATH_SEPARATOR_S, 0);
 #endif
